src/cpr_data_access/models/__init__.py

```diff
@@ -1157,10 +1183,19 @@ def to_huggingface(
                 citation=citation or "",
             )
 
+        mapping = {
```

Comment:
Area of note @kdutia

Reply:
@harrisonpim

Reply:
what's going on with these `plain_urls`?

Reply:
The mapping from a Url object seemed to fail, not entirely sure why the pydantic update would have caused that!